website/src/pages/v3/migration/migration-from-yoga-v2.mdx

```diff
@@ -99,29 +100,15 @@ Now they are removed in favor `graphqlEndpoint` option. If you want to use `Yoga
 + import { createYoga } from 'graphql-yoga'
 - createServer({
 + createYoga({
--  endpoint: '/mypath',
+-  endpoint: '/graphql',
 -  graphiql: {
--    endpoint: '/mypath',
+-    endpoint: '/graphiql',
 -  },
-+  graphqlEndpoint: '/mypath',
-})
-```
-
-**You need to define `graphqlEndpoint` if your GraphQL endpoint is not `/graphql`**
```

Comment:
The idea here was to emphasize the need of `graphqlEndpoint` field if the endpoint is not "exact" `/graphql` in v3 compared to v2.

Reply:
But "_Removed `endpoint` and `graphiql.endpoint` options_" section already talks about this; if you used a different endpoint in v2, you'd need this too in v3 making that single section enough.